From 12a9deaaf85fd3011b0737e7e3c24e8e1a18a7b4 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Fri, 23 Jun 2017 12:43:06 +0800
Subject: [PATCH 1999/5242] MLK-15146: ASoC: fsl: add back fsl_hifi build

commit  a97c8a55df92c2142b8d6b8c78331c7472264286 from
https://source.codeaurora.org/external/imx/linux-imx.git

fsl_hifi is removed wrongly by commit a69a185aad7f ("MLK-15140-2:
ASoC: fsl: add machine driver for wm8524")

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/fsl/Makefile b/sound/soc/fsl/Makefile
index e441282..cbd90bf 100644
--- a/sound/soc/fsl/Makefile
+++ b/sound/soc/fsl/Makefile
@@ -14,6 +14,7 @@ obj-$(CONFIG_SND_SOC_P1022_RDK) += snd-soc-p1022-rdk.o
 # Freescale SSI/DMA/SAI/SPDIF Support
 snd-soc-fsl-acm-objs := fsl_acm.o
 snd-soc-fsl-asrc-objs := fsl_asrc.o fsl_asrc_dma.o
+snd-soc-fsl-hifi4-objs := fsl_hifi4.o
 snd-soc-fsl-sai-objs := fsl_sai.o
 snd-soc-fsl-ssi-y := fsl_ssi.o
 snd-soc-fsl-ssi-$(CONFIG_DEBUG_FS) += fsl_ssi_dbg.o
-- 
1.7.9.5

